### PR TITLE
Developer setup guide

### PR DESCRIPTION
**Pull request by udoheld**

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Build guide
+
+ChroMapper is developed with Unity **2019.3.0f6** in C#.
+
+This GitHub repository comes with the assets and scripts you need to easily open it up in Unity. Feel free to fork the repository if you plan on contributing to ChroMapper in any way. Even though ChroMapper is in closed alpha, if you are able to clone or fork and get the project working on your end, I'll allow you to use it as long as you're making contributions to ChroMapper. The setup is straight format
```
Comment (Caeden117) [inline BUILD.md:5]: `The setup is straight format` seems like an error. Perhaps you were looking for `The setup is straight forward.`?

File: BUILD.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Development Environment Setup
+* Clone the project from GitHub to your local work folder
+* Download and install [Unity Hub](https://unity3d.com/get-unity/download)
+* Activate your license within Unity Hub, most people should be eligible for a free Personal license
+* If you want to select your Visual Studio installation location install it upfront, otherwise the installer will put it into the default location
+* Install the correct Unity version mentioned in the header of this page from within Unity Hub in the "Installs" section. The project might not be using the latest release. In this case you should consult the "download archive" (from within Unity Hub). The installer will take care of the required dependencies as well.
+* Add the project in the "Projects" section. Select your main folder you cloned from GitHub.
+
+## Running the project
```
Comment (Caeden117) [inline BUILD.md:15]: This seems complex. You should be able to just run from the Unity editor just fine without having to do anything inside of Visual Studio.

File: BUILD.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+* Select "File" -> "Build and Run" within the IDE.
+* Warning "Missing Project ID" will appear. Ignore it and select "Yes". 
+* You can ignore the "Discord RPC error"
+
+## Repository layout
+
+The project works the master / development branch setup.
```
Comment (Caeden117) [inline BUILD.md:22]: Again, looks like there's an error here. Were you looking for `The project works with the master / development branch setup.`?

File: BUILD.md (hunk 2 above)
Comment (udoheld) [reply BUILD.md:15]: Well, I was referring to Unity as an IDE. I changed it to Unity so it's less ambiguous now.

**Comment by Caeden117** [comment]
Cool, thanks!

**Pull request merged**